src/components/VmIcon/index.js

```diff
@@ -9,8 +9,8 @@ import style from './style.css'
 
 const VmIcon = ({ icon, className, missingIconClassName }) => {
   if (icon) {
-    const data = icon.get('data')
-    const type = icon.get('type')
+    const data = icon.data
```

Comment:
consider `const VmIcon = ({ {data, type} = {}, ...`

Reply:
I was going for the minimal change.  Not sure if destructuring in the arguments for this one adds anything useful.  Maybe just compacting the code a bit?

Reply:
It looks ok with the destructuring, I'll make that change.